addons/point_of_sale/models/pos_config.py

```diff
@@ -250,7 +250,8 @@ def _compute_current_session(self):
         """If there is an open session, store it to current_session_id / current_session_State.
         """
         for pos_config in self:
-            session = pos_config.session_ids.filtered(lambda s: not s.state == 'closed' and not s.rescue)
+            session = pos_config.session_ids.filtered(lambda s: s.user_id.id == self.env.uid and \
```

Comment:
I would not change this just for the badge "Opened By", we probably can always display the badge 'Open By' when it is open.

Reply:
Well the things is that it was changed by this commit no so long ago: https://github.com/odoo/odoo/commit/4c4adf472453de65e45e71f0cbd22a36ef2ed0db

But I think the fix proposed here in `pos_discount` fixes your original issue without the need to change this. In other words, I reverted your fix and applied another :smiley: